GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/GpsDataCaptureService.java

```diff
@@ -46,12 +43,13 @@
     private LocationManagerListener locationManagerListener;
     private FusedLocationProviderListener fusedLocationProviderListener;
 
-    private static File gpxDataFolder;
+    private static File gpxFileFolder;
     private static File gpxFile;
 
     private static TextView gpsDataTextView;
     private static TextView gpsStatusTextView;
     private static String gpsStatus;
+    private static boolean isNewFile;
 
     public enum LocationApiVersion {FUSEDLOCATIONPROVIDERCLIENT, LOCATIONMANAGER}
```

Comment:
LocationApiVersion  or apiVersion is confusing name; usually version is an integer identifying some incremental changes.
here is probably apiChoice? or apiSelector or anything but version.

Reply:
Done.